Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.yml

```diff
@@ -1,181 +1,203 @@
+    outputs:
+    - contextPath: Arcanna.Event.arcanna_event
+      description: Full export for specified event
+  - name: arcanna-trigger-train
+    arguments:
+    - name: job_id
+      required: true
+      description: Job ID for which training will be triggered
+    - name: username
+      required: true
+      description: Username that will be assigned for train
+    outputs:
+    - contextPath: Arcanna.Train.status
+      description: Status of request ( OK if successful )
+      type: String
+    - contextPath: Arcanna.Train.error_message
+      description: Details if error on request
+      type: String
+    description: Trigger AI Train for specified Arcanna.ai Job
+  - name: arcanna-get-decision-set
+    arguments:
+    - name: job_id
+      required: true
+      description: Job ID for which to get decision labels set
+    outputs:
+    - contextPath: Arcanna.Event.decision_set
+      description: List of available labels for specified AI Job
+      type: List
+  - name: arcanna-send-event
+    arguments:
+    - name: job_id
+      description: An Arcanna running job_id
+    - name: event_json
+      required: true
+      description: json event for arcanna to inference.
+    - name: title
+      required: true
+      description: event title
+    - name: severity
+      description: event severity
+    - name: id_value
+      description: Value to use for ID Reference
     outputs:
     - contextPath: Arcanna.Event.event_id
-      description: Arcanna event id.
+      description: Arcanna event id
       type: Number
     - contextPath: Arcanna.Event.status
-      description: Arcanna ingestion status.
+      description: Arcanna ingestion status
       type: String
     - contextPath: Arcanna.Event.ingest_timestamp
-      description: Arcanna ingestion timestamp.
+      description: Arcanna ingestion timestamp
       type: date
     - contextPath: Arcanna.Event.error_message
-      description: Arcanna error message if any.
+      description: Arcanna error message if any
       type: String
     - contextPath: Arcanna.Event.job_id
       description: An Arcanna Job id used for sending.
       type: Number
-  - arguments:
-    - description: Arcanna Job Id.
-      name: job_id
-    - description: Arcanna generated unique event id.
-      name: event_id
+    description: Sends a raw event to Arcanna
+  - name: arcanna-get-event-status
+    arguments:
+    - name: job_id
+      description: Arcanna Job Id
+    - name: event_id
       required: true
-    description: Retrieves Arcanna Inference result.
-    name: arcanna-get-event-status
+      description: Arcanna generated unique event id.
     outputs:
     - contextPath: Arcanna.Event.event_id
-      description: Arcanna event id.
+      description: Arcanna event id
       type: String
     - contextPath: Arcanna.Event.ingest_timestamp
       description: Arcanna ingestion timestamp.
       type: String
-    - contextPath: Arcanna.Event.confidence_level
```

Comment:
So removing outputs from the yml also breaks backward compatibility. 
I would leave it as is and add the new values that return.

Reply:
re-added